crates/y-sweet/src/server.rs

```diff
@@ -440,37 +431,50 @@ async fn get_doc_as_update(
 async fn get_doc_as_update_deprecated(
     Path(doc_id): Path<String>,
     State(server_state): State<Arc<Server>>,
-    authorization: Option<TypedHeader<headers::Authorization<headers::authorization::Bearer>>>,
+    auth_header: Option<TypedHeader<headers::Authorization<headers::authorization::Bearer>>>,
```

Comment:
A lot of these changes are just renaming vars/args to better differentiate between an Authorization (an enum that indicates read/write access) and an auth header.